ppr-ui/src/views/mhrInformation/MhrInformation.vue

```diff
@@ -440,13 +445,27 @@ export default defineComponent({
     }
 
     const parseSubmittingPartyInfo = (): void => {
-      const submittingParty = {
-        businessName: localState.accountInfo.name,
-        address: localState.accountInfo.mailingAddress,
-        emailAddress: localState.accountInfo.accountAdmin.email,
-        phoneNumber: localState.accountInfo.accountAdmin.phone,
-        phoneExtension: localState.accountInfo.accountAdmin.phoneExtension
-      } as SubmittingPartyIF
+      var submittingParty = {} as SubmittingPartyIF
```

Comment:
The main reason we use `let` over `var` is that you have more granular control of the scope of the blocked variable. 
ie, by using var here, it is scoped to the entirety of the function. Although that may not be an issue here, in more complex functions it very well could be. 
It just best practices to use `let` to scope your variable to the wrapping `{}` vs the function block as a whole.

Reply:
Makes sense... I made the change...